lib/data/schema.js

```diff
@@ -82,8 +82,11 @@ const _recurseFormFields = (instance, bindings, repeats, selectMultiples, envelo
       if (binding != null) {
         field.type = binding.type || 'unknown'; // binding should have a type.
         const prop = saveToAttr(binding);
-        if (prop)
+        if (prop) {
+          if (Array.from(repeats).find((repeat) => bindingPath.startsWith(repeat)))
```

Comment:
Does this work when repeat is inside a group? What would be the value of `bindingPath` and `repeat` when saveto on /group1/repeat1/somefield

Reply:
excellent question

Reply:
in a test i am about to add, the binding path might look like
`/data/children/child/possessions/toys/toy`
and the set of repeats like
`'/data/children/child',  '/data/children/child/possessions/toys'`
so it does find it and reject, even when it is inside a complicated nesting.